shell/platform/darwin/ios/framework/Source/platform_views_controller.mm

```diff
@@ -521,29 +666,19 @@ bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
   // offset.
   //
   // Note that the transforms are not applied to the clipping paths because clipping paths happen on
-  // the mask view, whose origin is always (0,0) to the flutter_view.
+  // the mask view, whose origin is always (0,0) to the _flutterView.
   transformMatrix.postTranslate(-clipView.frame.origin.x, -clipView.frame.origin.y);
 
-  embedded_view.layer.transform = GetCATransform3DFromSkMatrix(transformMatrix);
+  embeddedView.layer.transform = GetCATransform3DFromSkMatrix(transformMatrix);
 }
 
-// Composite the PlatformView with `view_id`.
-//
-// Every frame, during the paint traversal of the layer tree, this method is called for all
-// the PlatformViews in `views_to_recomposite_`.
-//
-// Note that `views_to_recomposite_` does not represent all the views in the view hierarchy,
-// if a PlatformView does not change its composition parameter from last frame, it is not
-// included in the `views_to_recomposite_`.
-void PlatformViewsController::CompositeWithParams(int64_t view_id,
-                                                  const EmbeddedViewParams& params) {
+- (void)compositeView:(int64_t)viewId withParams:(const flutter::EmbeddedViewParams&)params {
```

Comment:
the old comments are missing?

Reply:
Ah nice catch -- my intent was to move all comments to the prototypes in the `@interface` declaration but I only did the properties. Will do.

Reply:
Added the comments on the ones that were missing. Note that some of the missing comments have moved to the prototypes in the header file and the cases where the only comment was "Visible for testing" have been removed since they're now in a `Testing` category declared on `FlutterPlatformViewsController`.